src/util/DeviceFingerprint.js

```diff
@@ -10,11 +10,14 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-/* eslint complexity: [2, 7] */
+/* eslint complexity: [2, 8] */
 
 define(['vendor/lib/q', 'okta/jquery'], function (Q, $) {
 
   return {
+    _isMessageFromCorrentIframe: function($iframe, event) {
```

Comment:
nit: Did you mean`Current` instead of `Corrent`? Either way, what if we just generalize the method name to `_isMessageFromCorrectSource`